### ISSUE TITLE
Request: Documentation for writing modules

### ISSUE THREAD

**Issue by trondarild** [ikaros-project/ikaros-3#38]
It would be useful to either have a MD document somewhere, or a page in the wiki that details all the details of how to write a module, as exists for ikaros 2.

E.g. it can be hard to know the various ways it is possible to set the size of an output port, and which characters are invalid to have in the ikc file, like no parantheses in the description etc.

**Comment by ikaros-project** [comment]
Yes, this is badly needed and will be added as soon as the final touches to the WebUI are done.

**Comment by ikaros-project** [comment]
Basic info is available in the main ikaros wiki (not ikaros-3).
